### ISSUE TITLE
Discrepancy between printed training loss and plotted training loss

### ISSUE THREAD

**Issue by jamesheatonrdm** [davidezordan/ImageClassifier#1]
Why is the training loss that gets printed calculated as running_loss / print_every whereas the loss that gets plotted is running_loss / len(dataloader)
